libs/domain/user/address-form/address-form.component.ts

```diff
@@ -171,19 +171,29 @@ export class UserAddressFormComponent
     this.countryService.set(this.country);
   }
 
+  protected preventInvalidDisplay = (e: Event): void => {
+    e.stopImmediatePropagation();
```

Comment:
Probably better to remove event listener inside handler, after the first handling, to make the behavior stable, ie:

```
protected preventInvalidDisplay = (e: Event): void => {
    e.stopImmediatePropagation();

    this.form.removeEventListener(
        'invalid',
        this.preventInvalidDisplay,
        true
    );
};
```

Reply:
And also leave a comment about the purpose of this hack, and cover new functionality with tests

Reply:
I wanted to do it like that, but `invalid` does not bubble and only fires on elements inside the form, not the form itself, so removing inside the handler will not remove all of the listeners, just the first one.

At least that's what happened when I played with it.